src/main/java/com/epam/pages/main/AdminPage.java

```diff
@@ -26,7 +26,7 @@ public boolean checkAllElementsArePresent() {
         );
     }
 
-    public boolean checkUIofStudentsAndParentsSections() {
+    public boolean checkUIofChosenSectionSections() {
```

Comment:
Please change the method name to "checkUIOfChosenSection".

Reply:
done